parent/pom.xml

```diff
@@ -215,7 +215,7 @@
         <google-api-services-mail-version>v1-rev81-1.22.0</google-api-services-mail-version>
         <google-api-services-bigquery-version>v2-rev352-1.22.0</google-api-services-bigquery-version>
         <google-api-services-pubsub-version>v1-rev12-1.22.0</google-api-services-pubsub-version>
-        <google-cloud-pubsub-version>1.102.0</google-cloud-pubsub-version>
+        <google-cloud-pubsub-version>1.108.1</google-cloud-pubsub-version>
```

Comment:
I think that's the reason for removing the exclusions, but we need to double check

Reply:
Yes that is correct, see my comment above.